core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -138,27 +149,54 @@ public static void clearCache() {
     }
 
     /**
-     * Gets the UnderFileSystem instance according to its schema.
+     * Gets the {@link UnderFileSystem} instance according to its schema without specific ufs conf.
      *
      * @param path the file path storing over the ufs
      * @return instance of the under layer file system
      */
+    // TODO(binfan): Remove this method, currently it is only used in tests
     public static UnderFileSystem get(String path) {
       return get(path, null);
     }
 
     /**
-     * Gets the UnderFileSystem instance according to its scheme and configuration.
+     * Gets the {@link UnderFileSystem} instance according to its scheme and configuration.
      *
-     * @param path the file path storing over the ufs
+     * @param path the path of mount point
      * @param ufsConf the configuration object for ufs only
      * @return instance of the under layer file system
      */
-    public static UnderFileSystem get(String path, Object ufsConf) {
+    public static UnderFileSystem get(String path, Map<String, String> ufsConf) {
       Preconditions.checkArgument(path != null, "path may not be null");
 
       return UFS_CACHE.get(path, ufsConf);
     }
+
+    /**
+     * @return the instance of under file system for Alluxio root directory
+     */
+    public static UnderFileSystem getForRoot() {
+      String ufsRoot = Configuration.get(PropertyKey.MASTER_MOUNT_TABLE_ROOT_UFS);
+      Map<String, String> ufsConf = Configuration.getNestedProperties(
+          PropertyKey.MASTER_MOUNT_TABLE_ROOT_OPTION);
+      return get(ufsRoot, ufsConf);
+    }
+
+    /**
+     * @param path journal path in ufs
+     * @return the instance of under file system for Alluxio journal directory
+     */
+    public static UnderFileSystem getForJournal(URI path) {
+      return getForJournal(path.toString());
+    }
+
+    /**
+     * @param path journal path in ufs
+     * @return the instance of under file system for Alluxio journal directory
+     */
+    public static UnderFileSystem getForJournal(String path) {
```

Comment:
Is `getForJournal` just a shortcut for `get` with `null` options? It is not specific to journaling at all, right? Why not overload `get()`?